INV4/pallet-inv4/src/lib.rs

```diff
@@ -172,6 +174,9 @@ pub mod pallet {
 
         /// Weight info for dispatchable calls
         type WeightInfo: WeightInfo;
+
+        /// Weight to fee conversion provider, from pallet_transaction_payment
```

Comment:
This comment doesn't match the line below it

Reply:
I mean to reference that the runtime config intended here is byte to fee in the way it is configured for pallet_transaction_payment in tinkernet